src/Metadata/Property/Factory/ExtractorPropertyMetadataFactory.php

```diff
@@ -48,9 +48,15 @@ public function create(string $resourceClass, string $property, array $options =
             }
         }
 
+        try {
+            $isInterface = (new \ReflectionClass($resourceClass))->isInterface();
+        } catch (\ReflectionException $e) {
+            $isInterface = false;
+        }
+
         if (
```

Comment:
`!$isInterface` costs less, you can check it first.

Reply:
Fixed 👍